### DISCUSSION TITLE
nuxt generate did not render html tags

### DISCUSSION THREAD

**Discussion by itsalimanuel** [nuxt/nuxt#16501] [Questions]
Hello guys , after finishing my project i run `nuxt generate` , the files created in dist, all normal but the home page for examples it's an html file but inside it have scripts not html tags, why ?
also in my nuxt.config.js i used: 
`  target: 'static',
  ssr: false,
  generate: {
    fallback: true
  },`
  but if i remove them, the website will stop and get error 
  
![image](https://user-images.githubusercontent.com/44509661/198369400-faea141e-65af-4926-b125-f82c4cf8222f.png)

all the pages now like that, any way to fix it ?

**Comment by harlan-zw** [comment]
Related issue: https://github.com/nuxt/framework/issues/5492 ?

**Reply by TheAlexLichter** [reply]
I think it is more about general SSG

**Comment by TheAlexLichter** [comment]
Because you turned of SSR, the generated sites will not have any content in the HTML. You want to use `nuxt generate` + `ssr: true` (default) to generate your static site indeed. Also `target: 'static'` is obsolete ☺️

**Reply by itsalimanuel** [reply]
I see , but now for example i got another problem , now the nested links not working normaly, for example before this link works normally :
https://admin.mwt-group.org/works/2318962b-4899-4876-a8f0-d7ec2a06f7fb
now not!

my code is :

```javascript

async getProject() {
      let id = this.$route.params.slug
      await this.$axios.get('https://cloud.squidex.io/api/content/username/works/' + id).then(res => {
        if (res.data.data) {
          let data = res.data.data
          this.project = {
            workDescription: data.workDescription,
            workImage: data.workImage.iv[0],
            workLink: data.workLink,
            workName: data.workName,
            workSlider: data.workSlider,
            workType: data.workType,
          }
        }
      })
    }

**Reply by TheAlexLichter** [reply]
For subpages that are not pre-rendered, the fallback files (usually 200.html) should be used. You might have to configure your webserver to server them properly.

See also https://github.com/nuxt/framework/issues/5003
